File: GuardianMonsters/inventory/src/main/java/de/limbusdev/guardianmonsters/inventory/ui/widgets/items/KeyItemDetailViewWidget.java
package main.java.de.limbusdev.guardianmonsters.inventory.ui.widgets.items;

import com.badlogic.gdx.scenes.scene2d.ui.Skin;
import com.badlogic.gdx.utils.ArrayMap;

import de.limbusdev.guardianmonsters.guardians.items.Inventory;
import de.limbusdev.guardianmonsters.guardians.monsters.AGuardian;

/**
 * Created by georg on 20.02.17.
 */

public class KeyItemDetailViewWidget extends main.java.de.limbusdev.guardianmonsters.inventory.ui.widgets.items.ItemDetailViewWidget
{

    public KeyItemDetailViewWidget(Skin skin, Inventory inventory, ArrayMap<Integer, AGuardian> team) {
        super(skin, inventory, team);

        getUse().remove();
        getDelete().remove();

    }
}

File: GuardianMonsters/inventory/src/de/limbusdev/guardianmonsters/inventory/ui/widgets/items/KeyItemDetailViewWidget.kt
package de.limbusdev.guardianmonsters.inventory.ui.widgets.items

import de.limbusdev.guardianmonsters.guardians.items.Inventory
import de.limbusdev.guardianmonsters.guardians.monsters.Team

/**
 * KeyItemDetailViewWidget displays information about key items. It has no use or delete buttons,
 * since key items are not usable and must not be thrown away.
 *
 * @author Georg Eckert 2018
 */

class KeyItemDetailViewWidget
(
        inventory: Inventory,
        team: Team
)
    : ItemApplicationWidget(inventory, team)
{
    init
    {
        // Key items cannot be "used" or "deleted", for that reason the buttons will be removed
        use.remove()
        delete.remove()
    }
}
